src/core/dep-graph.ts

```diff
@@ -138,6 +138,38 @@ class DepGraphImpl implements types.DepGraphInternal {
     return count;
   }
 
+  public isSame(other: types.DepGraph): boolean {
+    const sortByNodeId = (nodeA: any, nodeB: any) => nodeA.nodeId.localeCompare(nodeB.nodeId);
```

Comment:
let's also compare the packages (and the the `pkgInfo` on each)

Reply:
done